src/sentry/plugins/base/manager.py

```diff
@@ -46,40 +42,7 @@ def all(self, version=1):
             yield plugin
 
     def _register_work_batches(self, class_path):
-        # TODO: Not caching this load for now for simplicity, fix that in the child class
```

Comment:
Good catch - should have written something about this. It's OK that this is removed - it will be addressed in future commits. I assume (some time since I did this) that I got an issue while pushing in the patch and decided to resolve it by removing this legacy code.